content/en/docs/how-to/registry-authentication.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Authenticate with OCI-compliant registries"
+description: "How to use notation to authenticate with OCI-compliant registries"
+type: docs
+weight: 1
+---
+
+You can use `notation` to authenticate to an OCI-compliant registry. This is useful for pushing and pulling signed artifacts, as well as inspecting artifacts in the registry.
+
+
+## Use `notation login` to authenticate to an OCI-compliant registry
+
+To authenticate to an OCI-compliant registry, use the `notation login` command with your registry, username, and password. For example:
+
+```console
+notation login -u <username> -p <password> <registry>
```

Comment:
Using `notation login <registry>` only returns an odd (and not very friendly) error. This is inconsistent with other tools like `docker login` or `oras login`. We should clarify this because the error message is quite unclear whether those params need to be provided or there is an issue on the server side :(

Reply:
@toddysm This feedback seems like its related to the `notation` CLI and not the docs. Was there something you wanted me to change in this PR?

Reply:
I guess no. We already have a PR for fixing the issue.